docs/api/v1.24.md

```diff
@@ -4823,9 +4823,9 @@ Return information on the service `id`.
 
 #### Update a service
 
-`POST /services/(id or name)/update`
+`POST /services/(id)/update`
 
-Update a service. When using this endpoint to create a service using a
+Update a service with full ID. When using this endpoint to create a service using a
```

Comment:
I insist that we should have "full ID". Here is my reason that in this endpoint we can not use ID prefix to update a service. While other endpoints like 
```
DELETE /services/(id or name)
```
We can use prefix ID to delete a service.

So, there is still some difference.